api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1139,99 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Subscriptions allow applications to easily scale the number of
+      clients by managing consumed event offsets and distributing load between instances.
+      The key properties that identify subscription are 'owning_application', 'event_types' and 'use_case'.
+      It's not possible to have two different subscriptions with these properties being the same.
+    properties:
+      id:
+        type: string
+        description: |
+          Id of subscription that was created. Is generated by Nakadi, should not be specified when creating
+          subscription.
+        format: uuid
+        example: '105a76d8-db49-4144-ace7-e683e8f4ba46'
+      created_at:
+        type: string
+        description: |
+          Timestamp of creation of the subscription. Is generated by Nakadi, should not be specified when creating
+          subscription.
+        format: date-time
+        example: '1996-12-19T16:39:57-08:00'
+      owning_application:
+        type: string
+        example: 'price-service'
+        description: The id of application owning the subscription.
+      event_types:
+        type: array
+        items:
+          type: string
+        description: |
+          EventTypes to subscribe to.
+          The order is not important. Subscriptions that differ only be the order of EventTypes will be considered
+          the same and will have the same id.
+          * Currently only subscription to a single EventType is supported. Subscriptions with more than one EventType
+          in event_types property will be rejected.
+      use_case:
+        type: string
+        example: 'read-product-updates'
+        description: |
+          The value describing the use case of this subscription.
+          In general that is an additional identifier used to differ subscriptions having the same owning_application
+          and event_types.
+        default: 'none'
+      read_from:
+        type: string
+        enum:
+          - BEGIN
+          - END
+        description: |
+          Position to start reading events from. BEGIN - read from the oldest available event. END - read from the most
+          recent offset. Will be applied when the first client for this subscription starts consuming.
```

Comment:
@v-stepanov @rcillo @jkliff 
I don't agree with ''Will be applied when the first client for this subscription starts consuming"
Consider the following:
client starts consuming (server sends 1000 messages), but client did't answered anything to it (did not commit offsets). Than the moment of "consumption start" is not defined, but we already need to handle these offsets somehow.
I think that offsets must be calculated on time of creation of subscription.